src/truefoundry_sdk/types/flyte_task_template.py

```
@@ -2,20 +2,7 @@
-class FlyteTaskTemplate(UniversalBaseModel):
-    id: FlyteTaskId
-    custom: FlyteTaskCustom
-
-    if IS_PYDANTIC_V2:
-        model_config: typing.ClassVar[pydantic.ConfigDict] = pydantic.ConfigDict(extra="allow")  # type: ignore # Pydantic v2
-    else:
-
-        class Config:
-            smart_union = True
-            extra = pydantic.Extra.allow
+FlyteTaskTemplate = typing.Union[TruefoundryFlyteTaskTemplate, NativeSnowflakeFlyteTaskTemplate]
```

Ruff (0.15.9)

[warning] Use `X | Y` for type annotations

(UP007)